### ISSUE TITLE
[Usability Issue] Improve Accessibility of Language and Organisation Selection Feature with Higher Contrast Highlight Color or Clickable Option

### ISSUE THREAD

**Issue by kyendereta** [PalisadoesFoundation/talawa#1673]
**Is your feature request related to a problem? Please describe.**
Yes, the problem I experienced was that the highlight color made it difficult for me to select my language of choice.

**Describe the solution you'd like**
To improve accessibility, I suggest using a different color with higher contrast for highlighting or making the language selection option clickable, like in most apps. As a UX expert, I would be happy to implement these changes to ensure an optimal user experience.

**Describe alternatives you've considered**
I have not considered any alternative solutions at this time.

**Approach to be followed (optional)**
As the UX expert, I will review the current design and identify the most effective way to improve the accessibility of the language selection feature. I will then send to the development team to implement the chosen solution and conduct thorough testing to ensure its success.

**Additional context**
I found it challenging to see the highlighted language option due to the low contrast with the background color. I have attached a screenshot of the language selection page to illustrate the issue. Thank you for considering my proposal.

![talawagroup1](https://user-images.githubusercontent.com/56252221/225624592-3a578cea-ec04-41f4-b6c7-21855e035481.png)

**Comment by palisadoes** [comment]
Would you be writing the code to make these adjustments to the UI?

**Comment by kyendereta** [comment]
No, I will be bringing my UI/UX expertise to help with the design and user experience. However, I can work closely with the development team to ensure that the changes are implemented accurately and efficiently.

**Comment by palisadoes** [comment]
Start by opening a series of issues with your suggestions. We'll review each and assign them to people who ask if the change has merit. We'll close them if not.

**Comment by palisadoes** [comment]
Please start with this issue so that it can be assigned correctly.

**Comment by kyendereta** [comment]
Thank you for explaining the steps in simple terms. I understand that I need to open a series of issues with my suggestions about colors. I will make sure to be clear and specific in my suggestions, and start with the first issue mentioned so that it can be assigned correctly. Thanks again for your help!

**Comment by palisadoes** [comment]
It's best to submit a Figma design for review, for this. We need to know what your proposed vision is.

Doing this in isolation will lead to a worse outcome

**Comment by kyendereta** [comment]
I would like to apologize for opening a new issue without fully understanding the instructions or where to properly address my concerns. I realize that my actions may have caused confusion and inconvenience, and for that, I am truly sorry.

As you may recall from the recently closed issue, I have managed to create some pages for the Talawa mobile app. Here is the link to those pages: [[Talawa mobile prototype](https://www.figma.com/proto/vAlLuq05OUMj2Q5AXQx9TB/Talawa?node-id=51-14&scaling=scale-down&page-id=38%3A12&starting-point-node-id=133%3A509&show-proto-sidebar=1)]. Additionally, I wanted to inform you that I was able to utilize a plugin that translates the designs to Flutter code, as was mentioned in the previous issue.

**Comment by palisadoes** [comment]
We really need to see what you think all the screens would look like. 

1. Doing this for just logging in will create a good user experience as the login will be one design and the rest of the app will have another.
2. Then we'll need to see how this would work in both light and dark modes.

A UI/UX redesign needs to be done as a series of issues as part of a complete project. Doing this without a master plan isn't a good approach.

Please submit this as a project idea to either our Outreachy and/or GSoC internship programs. It would be interesting to see your entire thought process.

Closing.

**Comment by kyendereta** [comment]
I wanted to provide an update on the project that we have been discussing. I completed an intensive report and submitted it to the Outreachy internship program outlining what I have been able to achieve thus far and what I plan to do moving forward. I would be more than happy to discuss these findings with you and explore future possibilities.

In my report, I have highlighted the importance of creating a good user experience for the app and how we can maximise utilising accessibility based on the target audience of the app among other important things.
